test/test_manifest.json

```diff
@@ -18,6 +18,12 @@
        "type": "eq",
        "renderTaskOnContinue": true
     },
+    {  "id": "issue13130",
+       "file": "pdfs/issue13130.pdf",
+       "md5": "318518299132fe3b52252ca43a69a23e",
+       "rounds": 1,
+       "type": "eq"
+    },
```

Comment:
You probably don't need to test *all* pages, hence adding `"firstPage": 2,` and `"lastPage": 2,` here would suffice as far as I can tell.